srcpkgs/python-pex/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'python-pex'
 pkgname=python-pex
-version=1.4.6
+version=1.5.2
 revision=1
-wrksrc="pex-${version}"
 noarch=yes
+wrksrc="pex-${version}"
 build_style=python-module
 pycompile_module="pex"
 hostmakedepends="python-setuptools python3-setuptools python-Sphinx python3-Sphinx"
-depends="python python-wheel"
+depends="python python-wheel<=32.0"
```

Comment:
Was this supposed to be `<=0.32.0`? I can see that that requirement is specified in version.py but enforcing it here would make it uninstallable I think. Any chance it will still work with the current wheel version?

Reply:
Wow, I found it strange that I was able to install it locally even with the `python-wheel<=32.0` dependency.

In fact, it seems it's not possible to run it anymore on the current `python-wheel` version, not even `pex --help` is working anymore. Should this package be marked as broken? Or should I just s/32.0/0.32.0/ and hope the users know how to downgrade `python-wheel`?

Reply:
I think a package that cannot be installed without manually downgrading another is better than a package the can be installed but will almost always not run after installing. Since the previous version was already built by most of the builders, it is already in the repository so I would prefer the second option. Let's hope upstream comes up with a fixed version soon.

Reply:
By preferring the second option, you mean "a package the can be installed but will almost always not run after installing"? So should I just remove the version requirement?

Reply:
I refered to your post, so setting the correct version requirement, making it uninstallable without manually downgrading the wheel package first.

Reply:
Great, done!